Paintroid/src/androidTest/java/org/catrobat/paintroid/test/junit/model/LayerTest.java

```diff
@@ -135,33 +135,6 @@ public void testMergeLayers() {
 		assertThat(firstLayer.getBitmap().getPixel(1, 1), is(Color.BLUE));
 	}
 
-	@Test
```

Comment:
Why did we delete this one?

Reply:
As we changed some stuff with the layers the Test tests functionality that isn't there anymore